docs/sources/alerting/configure-notifications/template-notifications/_index.md

```diff
@@ -13,55 +13,95 @@ labels:
     - cloud
     - enterprise
     - oss
-title: Configure notification messages
+title: Template notifications
 weight: 430
 refs:
-  use-notification-templates:
+  template-annotations-and-labels:
     - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/use-notification-templates/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/templates/
     - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/use-notification-templates/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/templates/
+  manage-notification-templates:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/manage-notification-templates/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/manage-notification-templates/
   reference:
     - pattern: /docs/grafana/
       destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/reference/
     - pattern: /docs/grafana-cloud/
       destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/reference/
-  using-go-templating-language:
+  examples:
     - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/using-go-templating-language/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/examples/
     - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/using-go-templating-language/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/examples/
 ---
 
-# Configure notification messages
+# Template notifications
+
+You can use notification templates to change the title, message, and format of notifications.
+
+Grafana provides a default template for notification titles (`default.title`) and one default template for notification messages (`default.message`). Both templates display common alert details.
+
+You can also create a notification template to customize the content and format of your notification messages. For example:
+
+- Personalize the subject of an email or the title of a message.
+- Modify text within notifications, like selecting or omitting certain labels, annotations, and links.
+- Format text with bold and italic styles, and add or remove line breaks.
+
+However, you cannot change HTML or CSS code to modify the visual appearance, or alter the structure of the data passed to notification templates.
```

Comment:
Good catch.  I agree and have discussed this with @tonypowa. 

This PR does not change this and just display the existing previous content. 

I think we are not doing our best to communicate what is supported and not, and the specific details. Specially important for Email templates, which I guess is the most used contact point.

If we think this is an important are that we should document better, I suggest handling this in a subsequent PR. I'll keep track of this.

Reply:
@aifraenkel Yes, I reduced this because:
1 - I wanted to give less visibility to the limitations. 
2 - I wanted to include these limitations within each specific contact point (webhook, slack, teams, etc.).

 https://github.com/grafana/grafana/pull/94069/commits/c9b8c43f117522a40100fee94a11b43aa0fafdb8 address this.
 
 - bring back a more detailed list of constraints.
 - document them in the specific contact points (webhook, slack, teams, etc.).
 
 I am merging this now. But please feel free to let me know if we should include more information.